jdk/src/share/classes/sun/security/ec/SunECEntries.java

```diff
@@ -23,6 +23,12 @@
  * questions.
  */
 
+/*
+ * ===========================================================================
+ * (c) Copyright IBM Corp. 2018, 2022 All Rights Reserved
```

Comment:
When first changing a file from upstream the first copyright year should be the current year.